a10_neutron_lbaas/tests/unit/vthunder/test_instance_manager.py

```diff
@@ -0,0 +1,272 @@
+# Copyright 2015,  A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import mock
+import sys
+
+import mocks
+
+import a10_neutron_lbaas.a10_exceptions as a10_ex
+import test_base
+
+# Figure out why this is necessary to make the mock work correctly.
+
+log_mock = mock.MagicMock()
+
+# This is here because I couldn't get mock to actually mock the classes
+# Get rid of this and I'll give you a cookie - MD
```

Comment:
No one wanted a cookie?  In seriousness, this is awful and mock should be able to handle this with `mock.patch` at the class level.  It works so don't fix what ain't broken, but if there's a better way I'm all ears.